src/components/Backdrop/Backdrop.jsx

```diff
@@ -1,9 +1,27 @@
 import React from 'react';
+import PropTypes from 'prop-types';
+import styles from './Backdrop.css';
 
 const Backdrop = (props) => {
-  return (
-    <div> Backdrop Component </div>
-  );
+  if (props.showModalShareSave) {
+    return (<div
+      className={styles.show}
+      onClick={props.modalShareSaveHandler}
+      role="presentation"
+    />);
+  } else if (props.showModalGallery) {
+    return (<div
+      className={styles.show}
+      role="presentation"
+    />);
+  }
+  return null;
+};
+
+Backdrop.propTypes = {
+  showModalGallery: PropTypes.bool.isRequired,
+  showModalShareSave: PropTypes.bool.isRequired,
+  modalShareSaveHandler: PropTypes.func.isRequired,
 };
```

Comment:
Good job on using propTypes!

Reply:
Thanks.  I still need add that to several components, but it's coming along.